README.md

```diff
@@ -26,30 +26,6 @@ cd base-image && docker build --no-cache -t tsovak/insolar-jepsen-base . && cd .
 ./run-test.py -i insolar-jepsen:latest
 ```
 
-After the test:
-
-```
-# To login to `jepsen-1` pod:
-ssh -o 'StrictHostKeyChecking no' -i ./base-image/id_rsa -p 32001 gopher@localhost
-
-# To attach a process running in background:
-tmux ls
-tmux attach -t insolard
-
-# To copy a file from `jepsen-1` pod:
-scp -o 'StrictHostKeyChecking no' -i ./base-image/id_rsa -P 32001 gopher@localhost:.bash_profile ./
```

Comment:
This was just example how to copy files from pod, return it. Maybe add to description that file .bash_profile used as example.